### PR TITLE
Add RSKIP383 - POWpeg migration age normalization

### PR DESCRIPTION
**Pull request by marcos-iov**

### CODE REVIEW

File: IPs/RSKIP383.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+|**Purpose**    |Fair |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+Everytime the POWpeg composition changes a migration takes place. This migration consists of three stages:
+- Preparation: A new POWpeg has been voted but the new POWpeg has no control yet.
+- Activation: The new POWpeg is now active but the funds from the retiring POWpeg have not started to migrate yet.
```
Comment (marcos-iov) [inline IPs/RSKIP383.md:27]: I'm not sure the activation is a `stage`, it's something that happens in an instant.

Perhaps the three stages could be
- Pre-activation (or pending)?
- Migration
- Standard? (Not sure about this name, couldn't think of anything better)

What do you think?

File: IPs/RSKIP383.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- Migration: The UTXOs from the retiring POWpeg are migrated to the active POWpeg
+
+Each of these stages have a certain duration set in Rootstock blocks to enable users and/or systems to act on them.
```
Comment (nathanieliov) [inline IPs/RSKIP383.md:30]:
```suggestion
Each of these stages has a certain duration set in Rootstock blocks to enable users and/or systems to act on them.
```

Comment (josedahlquist) [reply IPs/RSKIP383.md:30]: As we are talking about more than one stage, _they have_ is the correct spell

File: IPs/RSKIP383.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+purpose: Fair
+layer: Core
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          |383           |
+| :------------ |:-------------|
+|**Title**      |POWpeg migration age normalization |
```
[the remark this answers is not in the document]
Comment (josedahlquist) [reply IPs/RSKIP383.md]: Thanks for raising this, corrected the RSKIP name to Increase POWpeg activation age

File: IPs/RSKIP383.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+rskip: 383
+title: POWpeg migration age normalization
```
[the remark this answers is not in the document]
Comment (josedahlquist) [reply IPs/RSKIP383.md]: same as above

File: IPs/RSKIP383.md (hunk 1 above)
Comment (josedahlquist) [reply IPs/RSKIP383.md:27]: I'm renaming the stages as follows:
- Pre-activation
- Pre-migration
- Migration

Comment (nathanieliov) [reply IPs/RSKIP383.md]: New renaming looks good to me

**Review by nathanieliov — APPROVED** [review]

**Review by kelvinator07 — APPROVED** [review]

**Pull request merged**